spec/error_reporters/rollbar_spec.rb

```diff
@@ -41,6 +41,10 @@
     context "given an empty rack_env" do
       let(:rack_env) { {} }
 
+      it "expects rack_env to be a hash" do
+        assert_kind_of(Hash, rack_env)
```

Comment:
it is somewhat redundant, but it is also trying to be extra clear so that the test double is not replaced by some other type of object